chat/src/main/java/net/md_5/bungee/api/chat/ComponentBuilder.java

```diff
@@ -77,8 +112,9 @@ public ComponentBuilder append(String text, FormatRetention retention)
     {
         parts.add( current );
 
```

Comment:
How is anything below this line related to " Add API to append BaseComponent arrays to ComponentBuilder"?